Main.sublime-menu

```diff
@@ -13,19 +13,11 @@
                 [
                     {
                         "caption": "PackageResourceViewer",
-                        "children":
-                        [
-                            {
-                                "command": "open_file",
-                                "args": {"file": "${packages}/PackageResourceViewer/PackageResourceViewer.sublime-settings"},
-                                "caption": "Settings – Default"
-                            },
-                            {
-                                "command": "open_file",
-                                "args": {"file": "${packages}/User/PackageResourceViewer.sublime-settings"},
-                                "caption": "Settings – User"
-                            }
-                        ]
+                        "command": "edit_settings",
```

Comment:
Yes, unfortunatly, this a ST3 command only, i've just checked. I don't really understand, *do you want to create a branch for ST2 and ST3, or not*?